proposals/0024-opacity-micromaps.md

```diff
@@ -60,7 +60,6 @@ A new `RAY_FLAG` is proposed that allows ray tracing in shaders to force OMM
 
 Add a built-in `RAYTRACING_PIPELINE_FLAG_ALLOW_OPACITY_MICROMAPS` to the
 `RAYTRACING_PIPELINE_FLAG` flags in HLSL to enable OMM.
-Add built-in `RAY_FLAG_FORCE_OMM_2_STATE` to `RAY_FLAG` flags in HLSL.
```

Comment:
Why was this removed?

Reply:
Because it's repeated on line 97, and this section seems to focus on RAYTRACING_PIPELINE_FLAG